modules/util/copy.go

```diff
@@ -5,7 +5,7 @@
 package util
 
 import (
-	"github.com/unknwon/com"
+	"github.com/unknwon/com" //nolint:depguard
 )
 
 // CopyFile copies file from source to target path.
```

Comment:
If i remember correctly golang v1.16 did add some features to std lib

Reply:
If we can find a replacement, please edit on this PR directly, at the moment I can't 😂